crates/db_schema/src/impls/post.rs

```diff
@@ -111,14 +118,11 @@ impl Post {
   ) -> Result<Vec<Self>, Error> {
     let conn = &mut get_conn(pool).await?;
 
-    let perma_deleted = "*Permananently Deleted*";
-    let perma_deleted_url = "https://deleted.com";
-
     diesel::update(post.filter(creator_id.eq(for_creator_id)))
       .set((
-        name.eq(perma_deleted),
-        url.eq(perma_deleted_url),
-        body.eq(perma_deleted),
+        name.eq(DELETED_REPLACEMENT_TEXT),
+        url.eq(DELETED_REPLACEMENT_URL),
```

Comment:
Url is optional, you can set `None` here and get rid of `DELETED_REPLACEMENT_URL`